MediaPlayer/Times.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import kind from '@enact/core/kind';
+
+import css from './MediaPlayer.module.less';
+
+/**
+ * Agate-styled formatted time component.
+ *
+ * @class Times
+ * @memberof agate/MediaPlayer
+ * @ui
+ * @private
+ */
+const Times = kind({
+	name: 'Times',
+
+	styles: {
+		css,
+		className: 'times'
```

Comment:
Shouldn't this have `line-height: 36px`? is the font size correct based on GUI guide?
Also, the start time should align with the start of the slider and the end time should align with the end of the slider.

Reply:
Added a `line-height: 36px` and `font-size:28px` to correspond the GUI guide. 
I left a screenshot on Harmony ticket for the time position, could you check it?

Reply:
Yeah.. I don't know but now it seems right.